stamina-testkit/src/test/scala/stamina/testkit/ScalatestTestGenerationSpec.scala

```diff
@@ -11,48 +11,71 @@ class ScalatestTestGenerationSpec extends StaminaTestKitSpec {
 
   import TestDomain._
 
-  case class ItemPersister(override val key: String) extends Persister[Item, V1](key) {
+  case class ItemPersister[V <: Version: VersionInfo](override val key: String) extends Persister[Item, V](key) {
```

Comment:
The use case for which this change is introduced is when you actually have a  persister for `V1` and `V2` with a migration, but you want to test a sample for which you only have a `V2` instance (perhaps because it uses fields not even present in `V1` yet).

I think it'd make the test more clear/realistic if we could use such an `ItemPersister` here - would that be doable?